src/components/CreateComment.jsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from "react";
+import { postComment } from "../../utils/api";
+
+function CreateComment({ article_id, setCommentsList }) {
+
+    const [body, setBody] = useState("")
+    const [username, setUsername] = useState('grumpy19')
+    const [hasCommented, setHasCommented] = useState(false)
+
+    function handleSubmit(event) {
+        
+        event.preventDefault();
+        const commentToPost = {
+            author: username,
+            body: body,
+            votes: 0
+        }
+
+        setCommentsList((currComments) => [commentToPost, ...currComments])
+        setBody('')
+        setHasCommented(true)
```

Comment:
This will also lead to the user seeing 'Comment Posted' even if the posting failed.